kafka/src/main/java/com/metamx/tranquility/kafka/util/PropertiesHelper.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.metamx.tranquility.kafka.util;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
+import com.google.common.collect.Sets;
+import com.metamx.common.IAE;
+import com.metamx.common.logger.Logger;
+import com.metamx.tranquility.kafka.model.TopicSchemaPair;
+import org.apache.commons.lang.Validate;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Read and validate application settings from a properties file.
+ */
+public class PropertiesHelper
```

Comment:
For the kafka properties, what do you think about taking anything that starts with `kafka.` and making it a kafka consumer property? That'd allow people to customize kafka consumer behaviors without us having to provide explicit support for them.

The only thing that we would need to make sure of is that we turn off autocommit, even if the user has set it on.